.github/workflows/ci.yml

```diff
@@ -64,8 +64,6 @@ jobs:
                 build_py_project_in_conda_env
                 run_examples
 
-                python wave/wave-op-mpi.py --lazy
```

Comment:
@matthiasdiener Out of curiosity, what went wrong when attempting to run this? Also does https://github.com/inducer/grudge/commit/3e710a8c1ff85ad1ae1eeadabff6bcc8ce5887ca resolve this now?

Reply:
I didn't want to introduce a hard dependency on pytato, also, the latest released version of pytato doesn't seem to be able to run this code.

Reply:
I don't think we need to introduce a hard dependency on `pytato` for this, we can add the git repo to `requirements.txt`.

Reply:
Good idea, I added it back to the tests